tests/reboot/reboot_test.go

```diff
@@ -144,9 +79,15 @@ var _ = Describe("{RebootAllNodes}", func() {
 							err = Inst().V.WaitDriverUpOnNode(n, Inst().DriverStartTimeout)
 							Expect(err).NotTo(HaveOccurred())
 						})
+
+						Step("validate apps", func() {
```

Comment:
why are we validating all again? this is being validated before destroying

Reply:
check https://github.com/portworx/torpedo/blob/eabf6abccbe3620e0cc172e814a527f0bc93dac9/tests/common.go#L368-L380

Reply:
done